easybuild/easyconfigs/r/rclone/rclone-1.42-amd64.eb

```diff
@@ -0,0 +1,35 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - https://hprc.tamu.edu
+
+easyblock = 'CmdCp'
+
+name = 'rclone'
+version = '1.42'
+versionsuffix = '-amd64'
+
+homepage = 'https://rclone.org/'
+
+description = """
+ Rclone is a command line program to sync files and directories to and from
+ a variety of online storage services
+"""
+
+toolchain = {'name': 'system', 'version': ''}
+
+source_urls = ['https://downloads.%(name)s.org/v%(version)s/']
+sources = ['%(name)s-v%(version)s-linux%(versionsuffix)s.zip']
+checksums = ['7a623f60a5995f33cca3ed285210d8701c830f6f34d4dc50d74d75edd6a5bfa6']
+
+cmds_map = [('', '')]
```

Comment:
hm... `CmdCp` derives from `MakeCp` and adds `cmds_map`, so maybe it makes more sense to use `MakeCp` for this? Although I suppose you'd have to skip the `build` step...

Reply:
Agreed, this is a bit hackish...